back-end/manage.py

```
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Ruff (0.14.4)

Shebang is present but file is not executable

(EXE001)